components/Icon/Icon.css

```diff
@@ -12,18 +12,26 @@
   line-height: inherit;
 }
 
-/* Types */
+.pactH {
+  height: 1.1em;
+  width: calc(1.1em * 2.272727273);
+}
+
```

Comment:
Yeah I don't know why I didn't pull the var from https://github.com/PactCoffee/loggins/blob/master/components/Logo/Logo.css#L7